.github/workflows/build-and-publish.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         run: ./gradlew $PREVIEW_TASK
         
   maven_Release:
-    if: ${{ github.ref == 'ref/head/master' }}
+    if: ${{ github.ref == 'ref/heads/master' }}
```

Comment:
should this also be `refs/heads/`?

```suggestion
    if: ${{ github.ref == 'refs/heads/master' }}
```

Reply:
Ah yes, more missing S's. Thanks for the catch!